chain/network/src/config_json.rs

```diff
@@ -231,6 +238,69 @@ fn default_tier1_new_connections_per_attempt() -> u64 {
     50
 }
 
+/// Indicates if Tier1 is enabled
+fn default_tier1_enabled() -> bool {
+    true
+}
+
+/// Configuration for Tier1 network
+///
+/// This replaces the previous configuration that was in the experimental section.
+/// Migration is simple:
```

Comment:
I think it makes more sense to put the migration instructions in the release notes rather than the source code.